rskj-core/src/main/java/co/rsk/core/TransactionListExecutor.java

```diff
@@ -129,9 +127,6 @@ public Boolean call() {
             }
 
             logger.trace("tx[{}] executed", i + 1);
-
-            // No need to commit the changes here. track.commit();
```

Comment:
This comment was in the master branch, so maybe keep it?

Reply:
There is no explanation about why it doesn't need to be committed there.. We can keep it tho.